app/core/models/ocelot/package.scala

```diff
@@ -42,7 +42,8 @@ package object ocelot {
   val listOfintegerRegex: Regex = s"$commaSeparatedIntsPattern".r
   val anyIntegerRegex: Regex = "^[\\-]?\\d+$".r
   val EmbeddedParameterRegex: Regex = """\{(\d)\}""".r
-  val exclusiveOptionRegex: Regex = """\[exclusive\]""".r
+  val exclusiveOptionRegex: Regex = "\\[exclusive:([A-za-z0-9:,\\s\\-_]+)\\]".r
+  val exclusiveOptionHintRegex: Regex = "\\[exclusive:([^\\]]+)\\]".r
```

Comment:
think you only need a single regex here, the latter which allows any non ] char would work

Reply:
Done